src/core/contexts.ts

```diff
@@ -2,66 +2,19 @@ import { createContext } from 'react'
 import type { Context } from 'react'
 import type { Atom, Scope } from './atom'
 import { createStore } from './store'
+import type { Store } from './store'
 
-const createScopeContainerForProduction = (
-  initialValues?: Iterable<readonly [Atom<unknown>, unknown]>
-) => {
-  const store = createStore(initialValues)
-  return [store] as const
+export type ScopeContainer = {
+  s: Store
 }
```

Comment:
Keeping this a wrapper object, so that we can add some more properties in the future.